test/k8sT/KafkaPolicies.go

```diff
@@ -115,116 +97,86 @@ var _ = Describe("K8sValidatedKafkaPolicyTest", func() {
+			helpers.DefaultNamespace, appPods[empireHqApp], prodHqDeathStar)
+		Expect(err).Should(BeNil(), "EmpireHqApp cannot produce messages")
 
 		err = kubectl.ExecKafkaPodCmd(
-			helpers.DefaultNamespace, appPods[outpostApp], fmt.Sprintf(conOutDeathStar))
-		Expect(err).Should(BeNil(), "Failed to consume from outpost on topic deathstar-plans")
+			helpers.DefaultNamespace, appPods[outpostApp], conOutDeathStar)
+		Expect(err).Should(BeNil(), "OutpostApp cannot produce messages")
 
 		err = kubectl.ExecKafkaPodCmd(
-			helpers.DefaultNamespace, appPods[backupApp], fmt.Sprintf(prodBackAnnounce))
-		Expect(err).Should(BeNil(), "Failed to produce to backup on topic empire-announce")
+			helpers.DefaultNamespace, appPods[backupApp], prodBackAnnounce)
+		Expect(err).Should(BeNil(), "BackupApp cannot not announce")
 
 		err = kubectl.ExecKafkaPodCmd(
-			helpers.DefaultNamespace, appPods[outpostApp], fmt.Sprintf(prodOutAnnounce))
-		Expect(err).Should(BeNil(), "Failed to produce to outpost on topic empire-announce")
-
-		By("Apply L7 kafka policy")
-		eps1 := kubectl.CiliumEndpointPolicyVersion(ciliumPod1)
-		_, err = kubectl.CiliumPolicyAction(helpers.KubeSystemNamespace, l7Policy, helpers.KubectlApply, 300)
-		Expect(err).Should(BeNil(), "Failed to apply Kafka L7 policy")
-
-		By("Waiting for endpoint updates with L7 policy")
-		err = helpers.WaitUntilEndpointUpdates(ciliumPod1, eps1, 3, kubectl)
-		Expect(err).Should(BeNil())
-		epsStatus1 := helpers.WithTimeout(func() bool {
-			endpoints1, err := kubectl.CiliumEndpointsListByLabel(ciliumPod1, podFilter)
-			if err != nil {
-				return false
-			}
-			return endpoints1.AreReady()
-		}, "could not get endpoints", &helpers.TimeoutConfig{Timeout: 100})
-
-		Expect(epsStatus1).Should(BeNil(), "Cilium endpoint list timeout")
-		endpoints1, err := kubectl.CiliumEndpointsListByLabel(ciliumPod1, podFilter)
-		Expect(err).Should(BeNil(), "Failed to get cilium endpoint list")
-
-		policyStatus1 := endpoints1.GetPolicyStatus()
-
-		/*
-			Kafka multinode setup:
-
-			K8s1:
-				1. empire-backup
-				2. empire-hq
-				3. kafka-broker : ingress policy only
-
-			K8s2:
-			   1. zook
-			   2. empire-outpost-8888
-			   3. empire-outpost-9999
-		*/
-		By("Testing Kafka endpoint policy enforcement status on K8s1")
-		Expect(policyStatus1[models.EndpointPolicyEnabledNone]).Should(Equal(2))
-		// Only the kafka broker app should have ingress policy enabled.
-		Expect(policyStatus1[models.EndpointPolicyEnabledIngress]).Should(Equal(1))
-		Expect(policyStatus1[models.EndpointPolicyEnabledEgress]).Should(Equal(0))
-		Expect(policyStatus1[models.EndpointPolicyEnabledBoth]).Should(Equal(0))
-
-		By("Testing endpoint policy trace status on kafka-broker node")
-
-		trace := kubectl.CiliumExec(ciliumPod1, fmt.Sprintf(
+			helpers.DefaultNamespace, appPods[outpostApp], prodOutAnnounce)
+		Expect(err).Should(BeNil(), "OutpostApp cannot not annouce")
```

Comment:
`OutpostApp cannot not annouce` does not make a lot of sense to me.
Maybe change to `Failed to produce to outpost on topic empire-announce`